iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -63,33 +69,37 @@ private static void logError(Validable validable, String message, ReportNode rep
         LOGGER.error("{}{}", validable.getMessageHeader(), message);
     }
 
-    private static void throwExceptionOrLogError(Validable validable, String message, boolean throwException, ReportNode reportNode) {
-        if (throwException) {
+    private static void throwExceptionOrLogError(Validable validable, String message, LogLevel throwException, ReportNode reportNode) {
+        if (throwException.equals(LogLevel.THROW_EXCEPTION)) {
             throw new ValidationException(validable, message);
         }
-        logError(validable, message, reportNode);
+        if (throwException.equals(LogLevel.LOG_ERROR)) {
+            logError(validable, message, reportNode);
+        }
     }
 
     public static void throwExceptionOrLogError(Validable validable, String message, ValidationLevel validationLevel, ReportNode reportNode) {
-        throwExceptionOrLogError(validable, message, validationLevel == ValidationLevel.STEADY_STATE_HYPOTHESIS, reportNode);
+        throwExceptionOrLogError(validable, message, checkValidationLogLevel(validationLevel), reportNode);
```

Comment:
Following the proposed name change, replace `checkValidationLogLevel` by `checkValidationActionOnError`

Reply:
Done